kms-codecommit/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "region" {
```

Comment:
IMHO you should define "description" and "type" and leave off "default" because none of these  defaults except "owner_tag" actually work so if you wing it you get a bogus Terraform run instead of a query which tells you to pay attention.

Reply:
OK, will do